src/resmom/catch_child.c

```diff
@@ -1365,9 +1366,10 @@ del_job_resc(job *pjob)
 			exit(99);	/* simulate crash */
 	}
 
-	/* remove PBS_NODEFILE - Mother Superior only has one */
+	/* remove PBS_NODEFILE - Mother Superior shall have one and the sister
+	moms too if the mom config secondary_pbs_nodefile is set to 1 */
 
-	if (pjob->ji_qs.ji_svrflags & JOB_SVFLG_HERE) {
+	if (pjob->ji_qs.ji_svrflags & JOB_SVFLG_HERE || secondary_pbs_nodefile) {
```

Comment:
i would think that a better name might be something like "gen_nodefile_on_sister_mom"?